wsclient/src/main/java/org/hpccsystems/ws/client/platform/ApplicationValueInfo.java

```diff
@@ -56,4 +62,13 @@ public ApplicationValue getRaw() {
         val.setValue(this.getValue());
         return val;
     }
+    
+    public org.hpccsystems.ws.client.gen.wsworkunits.v1_69.ApplicationValue getRawVersion6() {

```

Comment:
does Version6 refer to the 1.69, or the hpcc 6.0.0 ver?

Reply:
hpcc 6.x

Reply:
getRawHPCC6xVersion ?